tracing-journald/src/lib.rs

```diff
@@ -150,6 +152,18 @@ impl Subscriber {
         self
     }
 
+    /// Adds a field that will get be passed to journald with every log entry.
+    ///
+    /// This can for example be used to configure the syslog facility.
+    ///
+    /// See [Journal Fields](https://www.freedesktop.org/software/systemd/man/systemd.journal-fields.html)
+    /// and [journalctl](https://www.freedesktop.org/software/systemd/man/journalctl.html)
+    /// for more information.
+    pub fn with_journal_field(mut self, key: String, value: String) -> Self {
```

Comment:
This feature isn't be limited to special fields, so the documentation/naming shouldn't suggest as much. It's perfectly reasonable for someone to want to attach a custom field globally.

Reply:
Renamed `with_journal_field` to `with_custom_field` to indicate that.